Issue/#1587 (#1727)
Codecademy/docs: content/java/concepts/strings/terms/join/join.md

---

> Title: '.charAt()'

Reviewer: ```suggestion
Title: '.join()'
```

> Description: 'Returns the character at the given index in the string.'

Reviewer: ```suggestion
Description: 'Returns a new string composed of the elements joined together with the specified delimiter.'
```

Subjects:
  - 'Computer Science'
Tags:
  - 'Strings'

>   - 'Methods'

Reviewer 2: ```suggestion
  - 'Methods'
  - 'Characters'
```

CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> Returns a new string composed of the elements joined together with the specified delimiter.

Reviewer: ```suggestion
The **`.join()`** method returns a new string composed of the elements joined together with the specified delimiter.
```

## Syntax

```java

> String.join(CharSequence delimiter, CharSequence... elements);

Reviewer 2: ```suggestion
String.join(delimiter, element1, element2...elementN );
```

```

- `delimiter` (required): The delimiter that is used to join the elements.
- `elements`(required): The elements to be joined together.

## Example 1

Use `.join()` to create the string `"Hello World"` out of the given strings `"Hello"` and `"World"`:

```java
class Join {
  public static void main(String[] args) {
    String element1 = "Hello";
    String element2 = "World";

    String message = String.join(" ", element1, element2);
    System.out.println(message);
  }
}
```

The output would be:

```
Hello World
```

> 

Reviewer 2: I would recommend deleting the first example. A second example should only be added if it will showcase some other functionality.

Author: Yeah good point there, I agree with you!

> ## Example 2

Reviewer 2: ```suggestion
## Example 
```

> Use `.join()` to join the name of your friends to a single string, each name separated by a pipe:

Reviewer 2: ```suggestion
The example below uses `.join()` to concatenate several names with a pipe delimiter:
```

> ```java

Reviewer 2: ```suggestion
```java
// JoinPipe.java
```
